universum/modules/error_state.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+import inspect
+
+from ..lib.gravity import Module, Dependency
+
+
+class GlobalErrorState(Module):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)  # type: ignore
+        self.errors: List[str] = []
+
+
+class ErrorState(Module):
```

Comment:
I don't like the name of this class. If we keep it as-is, it implies that Launches 'is a' ErrorState, which is wrong.
I propose 'HasErrorState' or 'ErrorStorage', or even something like 'ErrorCognizant'

Reply:
Ok.